content/en/real_user_monitoring/reactnative/integrated_libraries.md

```diff
@@ -0,0 +1,28 @@
+---
+title: React Native Integrated Libraries
+kind: faq
+description: Collect RUM data from your React Native projects.
+further_reading:
+- link: https://github.com/DataDog/dd-sdk-reactnative
+  tag: GitHub
+  text: dd-sdk-reactnative Source code
+- link: https://www.datadoghq.com/blog/react-native-monitoring/
+  tag: Blog
+  text: Monitor your React Native applications
+- link: real_user_monitoring/explorer/
+  tag: Documentation
+  text: Learn how to explore your RUM data
+
+---
+## RUM Integrations
+
+### Integrate with ReactNavigation
+Datadog provides automatic integration for the [ReactNavigation](https://reactnavigation.org/) API (minimum supported version is `react-navigation/native@5.6.0`). Add the following in your source code:
```

Comment:
I'd suggest to use `React Navigation` which is the official name used on their website (https://reactnavigation.org/). I think it also feels a bit more straightforward to say we support *it* instead of *its API*.

```suggestion
Datadog provides automatic integration for [React Navigation](https://reactnavigation.org/) (minimum supported version is `react-navigation/native@5.6.0`). Add the following in your source code:
```